quipucords/scanner/tests_scanner.py

```diff
@@ -1,69 +0,0 @@
-#
-# Copyright (c) 2017 Red Hat, Inc.
-#
-# This software is licensed to you under the GNU General Public License,
-# version 3 (GPLv3). There is NO WARRANTY for this software, express or
-# implied, including the implied warranties of MERCHANTABILITY or FITNESS
-# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
-# along with this software; if not, see
-# https://www.gnu.org/licenses/gpl-3.0.txt.
-#
-"""Test the scanner capabilities."""
```

Comment:
Why delete? Didn’t this test scanjob some?

Reply:
I definitely may bring back the concept, but it doesn't really align with a class.  Part of the next PR.